Dockerfile

```diff
@@ -29,6 +31,18 @@ COPY api/ api/
 COPY controllers/ controllers/
 COPY pkg/ pkg/
 
+# Add kubectl tool
+RUN curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/${TARGETARCH}/kubectl"
+RUN chmod +x ./kubectl
+
+# Add helm chart DIR to get CRDs from it
+COPY deployment/network-operator chart
+# Update helm dependencies and copy CRDs from them
```

Comment:
nit: there is no update of helm dependencies here

Reply:
Correct. I updated the docstring. Update here will be required after we will upgrade the version of the NFD chart